### DISCUSSION TITLE
Thead border bottom

### DISCUSSION THREAD

**Discussion by krosoftware** [twbs/bootstrap#35217] [Q&A] [answered]
Hello
I downloaded v5.1.3 and created simple table:
```
<table class="table">
  <thead style="background-color: silver">
    <tr>
      <th scope="col">#</th>
      <th scope="col">First</th>
      <th scope="col">Last</th>
      <th scope="col">Handle</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">1</th>
      <td>Mark</td>
      <td>Otto</td>
      <td>@mdo</td>
    </tr>
  </tbody>
</table>
```
and I see black border-bottom (please check image here): https://ibb.co/vktgXqW
How to remove that border, please?
Thank you.

**Comment by adamplesnik** [comment, marked as answer]
Hi there, just set the SCSS variable, and you are good to go (assuming you are building the Bootstrap with your own variables).

` $table-group-separator-color: transparent`

More nasty solution, which doesn't require custom build:

```
/* To remove the line */
.table > :not(:first-child) {
  border-top-color: transparent;
}

/* To make the line thinner */
.table > :not(:first-child) {
  border-top-width: 1px;
}
```

**Reply by krosoftware** [reply]
👍

**Reply by ghost** [reply]
Using `$table-group-separator-color: transparent !default;` leave in finale css:

`.table > :not(:first-child) {
  border-top: 2px solid transparent;
}`

Is there a way to completely remove it without editing the source scss file ?

**Comment by adamplesnik** [comment]
You can remove all the lines from the table by using
```
<table class="table table-borderless">
  ...
</table>
```

**Reply by ghost** [reply]
Sorry my question wasn't clear. I use .scss file to override and add custom styles to bootstrap, but my question is:

Is it possible to remove a style from bootstrap without editing the source and without overwriting it with an empty rule ?

In this case i want to completely remove this part
`  > :not(:first-child) {
    border-top: (2 * $table-border-width) solid $table-group-separator-color;
  }`

from /bootstrap/scss/_tables.scss 

Hope this make sens
